crates/ide/src/rename.rs

```diff
@@ -131,6 +160,46 @@ pub(crate) fn will_rename_file(
     Some(change)
 }
 
+// FIXME: Allow usage with `extern crate`?
+fn alias_fallback(
+    sema: &Semantics<'_, RootDatabase>,
+    syntax: &SyntaxNode,
+    FilePosition { file_id, offset }: FilePosition,
+    new_name: &str,
+) -> RenameResult<Option<SourceChange>> {
+    match sema.find_node_at_offset_with_descend::<ast::UseTree>(syntax, offset) {
```

Comment:
Let's not try to handle macros here for now. That will make things more complicated and won't trivially work when the rename happens I reckon (adding a test for that otherwise would be good).

Reply:
Sure, decided to remove macro support ~I didn't knew `find_node_at_offset_with_descend` was supposed to work also with macros~.